src/game/models/monsters.js

```diff
@@ -634,89 +634,88 @@ t . Dynamo
       tokens: ['dynamo'],
     },
   },
+
+ /* #<{(|//////////////////////////////////////////////////////////////////////////////
+ Deeps Below
+
+ w . Plague Staff
+ s . Unholy
+ t . Pandora's Box Shard
+ */
+    {
+     id: 'colossal_spider',
+     name: 'Colossal Spider',
+     str: 150,
+     ref: 130,
+     acc: 150,
+     con: 130,
+     flow: 150,
+     level: 200,
+     prizes: {
+       exp: 200,
+       items: ['Spider Leg'],
+       tokens: ['Pandoras Box'],
+    },
+},
+      id: 'urork',
```

Comment:
This monster definition is giving a test error, it has no starting/opening curly bracket.